### PR TITLE
Added readme file for the hello world example

### PR DESCRIPTION
**Pull request by GloreenS**
### Related Links
Card [536](https://github.com/casper-network/docs/issues/536)

### Changes
Added readme file for the hello world assembly script contract.

### CODE REVIEW

File: smart_contracts/contracts_as/example/hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+# Hello World Assembly Script
+
+This assembly script accepts a message string and stores it in the calling account under the `special_value` NamedKey.
+
+**Usage**: This assembly script expects a runtime argument named `message` of type string.
+
+**Tests**: There are two tests available to test the Hello World assembly script. The `should_store_hello_world` test verifies the happy path, where a string *hello world* is saved under the `special_value` NamedKey. The `should_error_on_missing_runtime_arg` test verifies that an error is displayed when the runtime argument is missing. 
+The tests start by initializing the casper crates and creating a genesis account. Then the contract Wasm is loaded and the deploy_item object is created. The deploy_item object is passed to the execute_request. Finally, the execution engine is invoked to process the execute_request.
```
Comment (ACStone-MTS) [inline smart_contracts/contracts_as/example/hello-world/README.md]:
```suggestion
The tests start by initializing the Casper crates and creating a genesis account. Then the contract Wasm is loaded and the deploy_item object is created. The deploy_item object is passed to the execute_request. Finally, the execution engine is invoked to process the execute_request. 
```

File: smart_contracts/contracts_as/example/hello-world/README.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    }
+  }
+}
+```
+
+</details>
+
+:::note
+
+Make a note of the NamedKey uref hash in the faucet account details.
```
Comment (ACStone-MTS) [inline smart_contracts/contracts_as/example/hello-world/README.md]:
```suggestion
Make a note of the NamedKey URef hash in the faucet account details.
```

File: smart_contracts/contracts_as/example/hello-world/README.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+:::
+
+### View the value stored in the NamedKey
+
+You can view the value stored in the NamedKey by using the uref hash assigned to the NamedKey. To do this, we will first find the state root hash of the network using the following command:
```
Comment (ACStone-MTS) [inline smart_contracts/contracts_as/example/hello-world/README.md]:
```suggestion
You can view the value stored in the NamedKey by using the URef hash assigned to the NamedKey. To do this, we will first find the state root hash of the network using the following command:
```

File: smart_contracts/contracts_as/example/hello-world/README.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+  "result": {
+    "api_version": "1.0.0",
+    "state_root_hash": "6097c728c1af6179a491b5c3d143c49b032c9d6bb69be553729cb9f1489c3833"
+  }
+}
+```
+
+</details>
+
+Then use the uref hash and the state root hash to view the value stored in the NamedKey.
```
Comment (ACStone-MTS) [inline smart_contracts/contracts_as/example/hello-world/README.md]:
```suggestion
Then use the URef hash and the state root hash to view the value stored in the NamedKey.
```

**Review by ACStone-MTS — APPROVED** [review]
Content looks great to me! Just some small consistency suggestions.

File: smart_contracts/contracts_as/example/hello-world/README.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+        }
+      ]
+    }
+  }
+}
+```
+
+</details>
+
+:::note
```
Comment (TomVasile) [inline smart_contracts/contracts_as/example/hello-world/README.md]: typo?

**Review by TomVasile — CHANGES_REQUESTED** [review]

File: smart_contracts/contracts_as/example/hello-world/README.md (hunk 5 above)
Comment (GloreenS) [reply smart_contracts/contracts_as/example/hello-world/README.md]: updated style

**Pull request merged**